src/scripts/main.js

```diff
@@ -5,4 +5,65 @@ const people = require('./lib/people');
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+const table = document.querySelector('.dashboard');
+
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  for (const key in person) {
+    switch (key) {
+      case 'name':
+        createCell(person, 'name', row);
+        break;
+      case 'sex':
+        createCellGender(person, row);
+        break;
+      case 'born':
+        createCell(person, 'born', row);
+        break;
+      case 'died':
+        createCell(person, 'died', row);
+        break;
+    }
```

Comment:
Другий цикл однозначно зайвий, цю проблему можна було вирішити і без нього
```suggestion
    createCell(person, 'name', row);
    createCellGender(person, row);
    createCell(person, 'born', row);
    createCell(person, 'died', row);
```